Src/BootCamp.Chapter/Computer/DesktopComputer.cs

```diff
@@ -37,5 +37,15 @@ public Motherboard GetMotherboard()
         {
             return _motherboard;
         }
+
+        public DesktopComputer(Body body, Ram ram, Cpu cpu, Gpu gpu, HardDisk hard, Motherboard motherboard)
```

Comment:
You're supposed to assign field from what you pass in ctor. Don't hardcode-create new.

Reply:
Recoded.  Hopefully, closer to what is expected.